test/kubemark/resources/manifests/addons/kubemark-rbac-bindings/kubelet-binding.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1alpha1
+kind: ClusterRoleBinding
+metadata:
+  name: kubelet-cluster-admin
+  labels:
+    kubernetes.io/cluster-service: "true"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Yes you are right, kubelet requires just read access. I hadn't touched this file and let it stay like it was in e2e-rbac-bindings/kubelet-binding.yaml.
I really feel strange about two things here:
1. Why does the kubelet in e2e-rbac-bindings have cluster-admin role and not just the view role?
2. Kubemark runs after removing the kubelet-bindings.yaml file. How can this happen? Does it have something to do with it belonging to the system:nodes group? ([Ref](https://github.com/shyamjvs/kubernetes/blob/e1c6ad9fc4a095984789d9ed1f8ff79cb03619d3/test/kubemark/start-kubemark.sh#L155))
Am I missing something basic here?